src/app/pages/RuntimeAccountDetailsPage/hook.ts

```diff
@@ -93,5 +97,13 @@ export const useAccountEvents = (scope: SearchScope, address: string) => {
   )
   const { isFetched, isLoading, isError, data } = query
   const events = data?.data.events
-  return { isFetched, isLoading, isError, events }
+
+  if (isFetched && pagination.selectedPage > 1 && !events?.length) {
+    throw AppErrors.PageDoesNotExist
```

Comment:
Note, this does not apply to all events pages:
- err https://5c5850a6.oasis-explorer.pages.dev/mainnet/sapphire/address/0xC3ecf872F643C6238Aa20673798eed6F7dA199e9/events?page=999
- not err https://5c5850a6.oasis-explorer.pages.dev/mainnet/sapphire/tx/0x0c85e3dab5b7daac0e87663c37183235b8376ef3d4e7d1a5e2ad4c8b7b5523da?page=999

Reply:
Error added, but we handle errors differently. Sometimes we throw an error sometimes we render an empty state. Will create ClickUp task for Don to unify this.